tutorials/getting_started/item_widgets.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: documentation
+title: Pages - Item Widgets
+---
+
+# Custom Default Item Widgets
+
+In addition to customizing the Overview Page and the individual cards as discussed on the previous page, the way that the individual Point Items are shown, or whether or not they are shown at all, can be customized as well.
+In general the steps will include navigating to the Item in the Model tree or the Items Settings, clicking on "Add Metadata", and selecting "Default List Widget" to customize how the Item appears in the automatically generated cards on the Overview Page.
+One can also set the "Default Stand Alone Widget" and "Default Cell Widget" to change how an Item appears in other parts of MainUI.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+## Expressions
+
+Many portions of a widget can be configured to change dynamically based on the states of Items.
+This can be a powerful way to combine multiple Items into one widget (e.g. an oh-label widget showing the current state of a garage door that sends a command to a Switch Item to trigger the garage door opener when the widget is clicked).
+Common things one might use an expression for is to change an icon or color based on the state of an Item, hide a widget entirely if an Item isn't in a given state, colors of the widget elements, etc.
+For full details on expressions see the [Expressions docs]({{base}}/ui/building-pages.html#dynamically-configuring-components-with-expressions).
+
+Note that when working with Units of Masurement, the state of the Item needs to be parsed into a number for comparisons.
+For example -
+
+```javascript
+=(Number.parseFloat(items.MainFloor_Humidity.state) < 35) ? "red": "blue"
+```
+
+When creating anything but the simplest of expressions, the Widgets Expression Tester is a very helpful tool.
+This is the third tab of the Developer Sidebar (accessible from Developer Tools or by pressing ALT-SHIFT-d).
+Note that the sidebar will only appear if the browser window is wide enough.
+
+![expression tester](images/expression_tester.png)
+
+For complicated expressions, gradually build up the expression in this tool until it works as required and then paste it into the config for the widget.
+
+## Visibility
+
+Each custom defined widget has a "Visibility" and "Visible to" property.
+The "Visibility" option takes a boolean `true` or `false` (without quotes) or the result of a boolean expression to determine whether or not to render the widget.
+The "Visibile to" property controls which type of user can see the widget.
```

Comment:
Maybe make the emphasized part **bold**.
I think the do not get emphasized really with the quotation marks.

Reply:
I am using quotes to indicate that that is how the fields are literally labeled. I'm not intending them to be for emphasis.

Reply:
OK then.